src/overlays/Modal.tsx

```diff
@@ -43,6 +43,22 @@ const ModalFooter = (props: { actions: React.ReactNode[] }) => (
 )
 
 export const Modal = (props: ModalProps) => {
+  const DESKTOP_MIN_WIDTH = 767 // @screen md
```

Comment:
Surprised to see this javascript resizing piece and wondering if similar functionality could be incorporated into the CSS. Potentially, with some combination of @media and display:none. I'd envision this to also be a candidate for css variables so that consumers could keep the footer of the modal on mobile view if desired.